integration-tests/config.go

```diff
@@ -6,23 +6,40 @@ import (
 	"time"
 )
 
-// Attributes that are unique to a validator. Allows us to map (part of) the set of uints to
-// a set of viable validators
+type chainID string
+type validatorID string
+
+const (
+	// Strings mapped to validator configs, literals must be two characters long for network config
+	alice = validatorID("al")
```

Comment:
IMO you should just use the string "al" everywhere, not define a variable

Reply:
Because otherwise it makes it harder to debug by looking at what is inside the system